.htaccess

```diff
@@ -18,6 +18,7 @@ Options -Indexes
     Header set Access-Control-Allow-Origin: https://apisix.apache.org
     Header set Access-Control-Allow-Methods "*"
     Header set Access-Control-Allow-Headers "*"
+    Header set Content-Security-Policy "frame-src 'self' https://www.google.com https://app.netlify.com"
```

Comment:
Do I need to rollback this PR? https://github.com/apache/apisix-website/pull/1773 @SkyeYoung

Reply:
I think u can just delete the code, and ref the PR here.

Reply:
done